package.json

```diff
@@ -0,0 +1,27 @@
+{
+  "name": "nodementoring",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "nodemon --exec babel-node src/index",
```

Comment:
@lenon777, the program should be started from npm script via nodemon. For example, `npm run task1`. Each command should run separated homework